### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ jobs:
           override: true # default: false
 ```
 
+### Projects using this action
```
Comment (micnncim) [inline README.md]:
```suggestion
## Projects using this action
```

File: README.md [hunk 2]
```diff
@@ -43,6 +43,10 @@ jobs:
...
+
+* [Cake Website](https://github.com/cake-build/website)
```
Comment (micnncim) [inline README.md]:
```suggestion
- [Cake Website](https://github.com/cake-build/website)
```

**Comment by gep13** [comment]
@micnncim thanks for the review.  I have fixed up the suggestions.

**Comment by micnncim** [comment]
@gep13 thank you for the contribution including discussion!

**Pull request merged**